editors/vscode/src/common.ts

```diff
@@ -259,3 +292,36 @@ async function maybeSendStartupTelemetryEvent(
 
     telemetryLogger.logUsage("extension-activated", usageData);
 }
+
+const HELP_URL = "https://docs.slint.dev/latest/docs/slint/reference/";
```

Comment:
Could we make it `https://releases.slint.dev/{...version...}/docs/slint/reference` ?